controllers/apply-next/views/dashboard.njk

```diff
@@ -2,38 +2,159 @@
 {% from "components/content-box/macro.njk" import contentBox %}
 {% from "components/form-header/macro.njk" import formHeader with context %}
 {% from "components/form-fields/macros.njk" import formErrors, formField with context %}
+{% from "components/icons.njk" import iconBin %}
+{% from "./view-helpers.njk" import showSectionProgress with context %}
 
 {% extends "layouts/main.njk" %}
 
+{% set fieldMissing %}<span class="application-data-missing">not yet completed</span>{% endset %}
+
+{% macro getStartDate(dateParts) %}
```

Comment:
We possibly want to make this a view function like `formatDate` for this, but OK here for now.